bridge/opencensus/internal/oc2otel/span_context.go

```diff
@@ -14,9 +14,16 @@ func SpanContext(sc octrace.SpanContext) trace.SpanContext {
 	if sc.IsSampled() {
 		traceFlags = trace.FlagsSampled
 	}
+
+	tsOtel := trace.TraceState{}
+	for _, entry := range sc.Tracestate.Entries() {
+		tsOtel, _ = tsOtel.Insert(entry.Key, entry.Value)
+	}
```

Comment:
```suggestion
	entries := slices.Clone(sc.Tracestate.Entries())
	slices.Reverse(entries)

	tsOtel := trace.TraceState{}
	for _, entry := range entries {
		tsOtel, _ = tsOtel.Insert(entry.Key, entry.Value)
	}
```

Reverse oc entries before inserting to otel.